keel-sql/src/main/resources/db/changelog/20210316-update-env-artifact-constr.yml

```diff
@@ -0,0 +1,28 @@
+databaseChangeLog:
+  - changeSet:
+      id: drop-null-artifact-reference
+      author: emjburns
+      changes:
+        - sql:
+            dbms: mysql
+            sql: DELETE FROM environment_artifact_constraint WHERE artifact_reference is NULL
+  - changeSet:
+      id: non-null-artifact-reference
+      author: embjurns
+      changes:
+        - addNotNullConstraint:
+            tableName: environment_artifact_constraint
+            columnName: artifact_reference
+            columnDataType: varchar(150)
+  - changeSet:
+      id: new-unique-constraint
+      author: emjburns
+      changes:
+        - dropUniqueConstraint:
+            tableName: environment_artifact_constraint
+            constraintName: constraint_pk
+            uniqueColumns: environment_uid, type, artifact_version
+        - addUniqueConstraint:
```

Comment:
Shouldn't that be [`addPrimaryKey`](https://docs.liquibase.com/change-types/community/add-primary-key.html)?

Reply:
no, the primary key is actually the uuid. it's the unique constraint i needed to update. i was wrong when I pinged you earlier

Reply:
updated to not have pk in the name